### PR TITLE
Remove Tech Preview label from EAP 7.4

### PR DESCRIPTION
**Pull request by jkopriva**
Upversion of plugins

Signed-off-by: Josef Kopriva <jkopriva@redhat.com>

# Pull Request Checklist
## General
* Is this a blocking issue or new feature? If yes, QE needs to +1 this PR

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?

## Testing
* Are there unit-tests?
* Are there integration tests (or at least a jira to tackle these)?
* Is the non-happy path working, too?
* Are other parts that use the same component still working fine?

## Function
* Does it work?

### CODE REVIEW

**Comment by robstryker** [comment]
Please fix suggested changes from jeff.

**Review by jeffmaury — APPROVED** [review]
3.9.206 should be 3.9.200 but that's still fine that way.

**Comment by jkopriva** [comment]
@jkopriva I have changed the version to the correct: 3.9.200

File: as/itests/org.jboss.tools.as.ui.bot.itests/src/org/jboss/tools/as/ui/bot/itests/parametized/server/ServerAdaptersTest.java [hunk 1]
```diff
@@ -133,7 +133,7 @@ protected File getDownloadPath() {
 	private String getServerHome(String server) {
 		String homeFlag;
 		if (server.contains("WildFly")) {
-			String version = server.split(" ")[1];
+			String version = server.split(" ")[1].replaceAll("\\+","");
```
Comment (jeffmaury) [inline as/itests/org.jboss.tools.as.ui.bot.itests/src/org/jboss/tools/as/ui/bot/itests/parametized/server/ServerAdaptersTest.java:136]: Can you clarify or should this be removed ?

Comment (jkopriva) [reply as/itests/org.jboss.tools.as.ui.bot.itests/src/org/jboss/tools/as/ui/bot/itests/parametized/server/ServerAdaptersTest.java:136]: This is just quick fix for WildFly 24+ server adapter. I will rewrite it after the WF25 server adapter will be ready.

**Pull request merged**